### PR TITLE
Support comma separated strings for labels

### PR DESCRIPTION
**Pull request by ketenshi**
Clone of https://github.com/christianvuerings/add-labels/pull/5

Adds support for comma separated strings for the labels field so we can do this:

```
- id: label-maker
  run: |
     // generate labels list
     // labels="minor release,bug"
     echo "::set-output name=labels::$labels"

- uses: christianvuerings/add-labels@v1
   with:
   labels: ${{ steps.label-maker.outputs.labels }}
   env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const { GitHub, context } = require("@actions/github");
     const { owner, repo } = context.repo;
     const labels = core
       .getInput("labels")
-      .split("\n")
+      .split(/[\n,]/)
```
Comment (tahirmt) [inline index.js:16]: Why do we need to support comma separated strings?

Comment (tahirmt) [reply index.js:16]: Does adding this break support for doing something like

```
- uses: christianvuerings/add-labels@v1
   with:
   labels: |
      hello
      world
   env:
      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment (ketenshi) [reply index.js:16]: I explained that in the description 😛 

It's so we can dynamically generate the `labels` list. I tried to make the output `Regression\nCode complete` and it added 3 labels - `Regression`, `Code complete`, and `Regression\nCode Complete`

image

https://github.com/scorebet/sportsbook-ios/pull/2909

Comment (ketenshi) [reply index.js:16]: Old behaviour is unchanged from my tests

**Review by tahirmt — APPROVED** [review]

**Pull request merged**